### ISSUE TITLE
vaadin-maven-plugin fails with NoSuchMethodError when trying to delete an outdated node version

### ISSUE THREAD

**Issue by DJGummikuh** [vaadin/maven-plugin#135]
When trying to run mvn install, the vaadin-maven-plugin fails with
`Caused by: java.lang.NoSuchMethodError: org.apache.commons.io.FileUtils.delete(Ljava/io/File;)Ljava/io/File;`
presumably because it's pulling an outdated version of commons-io (2.8.0).
![2022-11-16_22-34](https://user-images.githubusercontent.com/5188486/202388640-d040aac2-3165-4ce2-929f-4178aaa3cf92.png)

This behavior only triggers if it finds an outdated version of node in ~/.vaadin/node and tries to update it (in my case from 14.15.4 to 16.18.0)

Tested with the latest version of the my-todo Getting started project, which pulls VAADIN 23.2.8.
Tested with Maven 3.6.0, 3.8.1 and 3.8.6
Tested with Java 11.0.1+13, 11.0.17+8, 17.0.1+12 and 17.0.5+8

Please see attached files for detailed Maven Output and effective pom.
[errorlog.txt](https://github.com/vaadin/maven-plugin/files/10029108/errorlog.txt)
[effectivepom.xml.zip](https://github.com/vaadin/maven-plugin/files/10029110/effectivepom.xml.zip)

**Comment by Artur-** [comment]
The version seems to be defined in https://repo1.maven.org/maven2/com/vaadin/vaadin-maven-plugin/23.2.8/vaadin-maven-plugin-23.2.8.pom

**Comment by cromoteca** [comment]
> The version seems to be defined in https://repo1.maven.org/maven2/com/vaadin/vaadin-maven-plugin/23.2.8/vaadin-maven-plugin-23.2.8.pom

Yes: https://github.com/vaadin/platform/blob/23.2/scripts/generator/templates/template-vaadin-maven-plugin-pom.xml#L41

Still there in `23.3`, not in `master`.
